README.md

```diff
@@ -1 +1,19 @@
-# server-deployment-practice
\ No newline at end of file
+LAB - Class 01
+
+Project: Server Deployment Practice
+Author: Garrett Cintron
+Links and Resources
+ci/cd [Github Actions](https://github.com/tiggerman120/server-deployment-practice/actions)
+back-end server url (when applicable)
+front-end application (when applicable)
```

Comment:
I would like to see those two heroku links in your README please!
I would also suggest making things like the title of this project as a header and then have sub-headers for different section. And add that extra space after each line so that you get that separation that you want!